media/css/firefox/welcome17.scss

```diff
@@ -40,6 +40,11 @@ $image-path: '/media/protocol/img';
     }
 }
 
+.c-main-subheading {
```

Comment:
**Nit (blocking/a11y):**  Place this under the dark mode preferences under line 195. It's showing dark text on dark-mode
![image](https://github.com/mozilla/bedrock/assets/42309026/b5a69796-c5b7-4bd4-8d3c-2810afedffab)